packages/core/addon/models/bard-request-v2/request.js

```diff
@@ -73,30 +73,38 @@ export default class Request extends Fragment.extend(Validations) {
       clonedRequest = this.toJSON();
 
     return store.createFragment('bard-request-v2/request', {
-      filters: clonedRequest.filters.map(filter =>
-        store.createFragment('bard-request-v2/fragments/filter', {
+      filters: clonedRequest.filters.map(filter => {
+        const newFilter = store.createFragment('bard-request-v2/fragments/filter', {
           field: filter.field,
           parameters: filter.parameters,
+          type: filter.type,
           operator: filter.operator,
           values: filter.values
-        })
-      ),
-      columns: clonedRequest.columns.map(column =>
-        store.createFragment('bard-request-v2/fragments/column', {
+        });
+        newFilter.source = clonedRequest.dataSource;
```

Comment:
Are we expecting the `filter` we are cloning not to have a `source` property?

Reply:
possibly yes.